packages/app-studio-toolkit/scripts/package-vsix.js

```diff
@@ -17,6 +17,22 @@ expect(pkgJson.main).to.equal("./dist/src/extension");
 pkgJson.main = "./dist/extension";
 writeJsonSync(pkgJsonPath, pkgJson, { spaces: 2, EOF: "\n" });
 
+// Ensure License and copywrite related files are part of the packaged .vsix
+const rootMonoRepoDir = resolve(__dirname, "..", "..", "..");
```

Comment:
should not this already happen here?
https://github.com/SAP/app-studio-toolkit/blob/main/package.json#L29-L31